plugin/plugin.go

```diff
@@ -70,17 +68,15 @@ func (p *CarMirrorPlugin) Init(env *plugin.Environment) error {
 }
 
 func (p *CarMirrorPlugin) Start(capi coreiface.CoreAPI) error {
-	log.Debugf("Start")
+	log.Debugw("CarMirrorPlugin", "method", "Start")
 
-	lng, err := carmirror.NewLocalNodeGetter(capi)
-	if err != nil {
-		return err
-	}
+	clientBlockStore := carmirror.NewKuboStore(capi)
```

Comment:
I don't know that a separate store is needed here.  I added when trying to debug the Get hang.